README.md

```diff
@@ -116,6 +112,14 @@ Additional resources:
 * [serverfull](https://github.com/asecurityteam/serverfull)
 * [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
 
+### Environment Variables
```

Comment:
Nit: It's not clear what the difference is between these environment variables and the ones listed above. Why are they in a separate list?

Reply:
so in docker-compose, there is defined 2 services, each with a list environment variables. The reason why they are a separate list is to reflect this. However, you are indeed correct, it's not clear at all